zuul.d/job.yaml

```diff
@@ -20,29 +18,24 @@
       - .coveragerc
       - LICENSE
       - OWNERS*
-
-- job:
-    name: tcib-podified-multinode-edpm-deployment-crc
-    parent: podified-multinode-edpm-deployment-crc
-    irrelevant-files: *if
     vars: &edpm_vars
-      cifmw_update_containers_registry: "{{ content_provider_registry_ip }}:5001"
-      cifmw_update_containers_org: "podified-antelope-centos9"
-      cifmw_update_containers_tag: "{{ content_provider_dlrn_md5_hash }}"
-      cifmw_update_containers_openstack: true
       cifmw_run_test_role: test_operator
       cifmw_run_tempest: true
-      cifmw_test_operator_tempest_registry: "{{ content_provider_registry_ip }}:5001"
+      cifmw_test_operator_tempest_registry: "{{ content_provider_os_registry_url | split('/') | first }}"
```

Comment:
just noticed, we had port before is it no longer required here? 
```
:5001
```

Reply:
@marios correct, It is no longer required. From successful result, https://github.com/openstack-k8s-operators/tcib/pull/179#issuecomment-2188280866.
 We can see meta content provider will return `content_provider_os_registry_url` to the [edpm zuul inventory](https://logserver.rdoproject.org/79/179/a0db69a8e875ce5e3454722e3c61cf6f16101805/github-check/tcib-podified-multinode-edpm-deployment-crc/5940ff6/zuul-info/inventory.yaml).
```
content_provider_os_registry_url: 38.102.83.3:5001/podified-antelope-centos9
```
So we no longer need to provide port here.

Reply:
ack thanks